pkg/api/login.go

```diff
@@ -102,12 +102,13 @@ func LoginPost(c *middleware.Context, cmd dtos.LoginCommand) Response {
 	}
 
 	authQuery := login.LoginUserQuery{
-		Username: cmd.User,
-		Password: cmd.Password,
+		Username:  cmd.User,
+		Password:  cmd.Password,
+		IpAddress: c.Req.RemoteAddr,
```

Comment:
Will this pick up up the `X-Forwarded-For` header if available?

Reply:
it should (I hope so at least)

Reply:
According to macaron source code it does: https://github.com/Unknwon/macaron/blob/master/context.go#L133